lib/client.js

```diff
@@ -119,11 +120,13 @@ Client.prototype._callSolrServer = function (
       throw new Error(error);
     } finally {
       const t1 = performance.now();
-      console.log("solr-node:_callSolrServer performance time", {
-        milliseconds: t1 - t0,
-        requestFullPath,
-        fetchOptions,
-      });
+      if(showLogs === "true"){
```

Comment:
I find this SOLR_NODE_LOGS a bit too vague... is it all logs? just perfomance time logs?

Reply:
Fair enough point. It's prefixed with the SOLR_NODE because it's set in the P1 project. I'm not sure what else to call it other than SOLR_NODE_SHOW_CONSOLE_LOGS. Any suggestions?